### PR TITLE
Set a suitable title for the website

### LINKED ISSUE #28: set a suitable title for the website
Currently, it shows the template name `Bare ....`

### PR DESCRIPTION
**Pull request by palak2001**
https://palak2001.github.io/CodeFemme/

### CODE REVIEW

**Comment by Anupam-dagar** [comment]
I didn't understand your PR. The issue says something, the PR title says something and in the changes, I can see something else.

**Comment by palak2001** [comment]
Fixes#28
https://palak2001.github.io/CodeFemme/

**Comment by palak2001** [comment]
I resolved the conflicts but I am not sure whether they are saved to PR or are present only in my repo

File: index.html [hunk 1]
```diff
@@ -8,7 +8,9 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
-    <title>Bare - Start Bootstrap Template</title>
+
+  <title>Bare - Start Bootstrap Template</title>
+  <title>CodinGirl</title>
```
Comment (Anupam-dagar) [inline index.html]: There should be only one title.

**Pull request merged**